darshan_logs/release_logs/README.md

```diff
@@ -0,0 +1,11 @@
+This directory contains logs generated for every 3.x version of Darshan.
+These logs are generated as part of the release process and are intended
+to ensure Darshan maintains backwards compatibility for parsing log files
+generated by older versions of the library.
+
+These test logs are generated with the mpi-io-test benchmark that is
+distributed as part of Darshan's internal regression testing framework
+(darshan-test/regression/test-cases/src/mpi-io-test.c). The benchmark
+is ran using 4 processes and uses all default parameters to mpi-io-test.
```

Comment:
This is really piddling, but "The  benchmark is ran" sounds funny to me :)  Maybe "The benchmark is executed"?

Reply:
no worries, i just pushed a fix for this :)